gdal/COMMITTERS

```diff
@@ -67,6 +67,9 @@ rcoup            Robert Coup                     OGR: FGDB
 dzwarg           David Zwarg                     PostGIS Raster
                  dzwarg@azavea.com
 
+ilucena          Ivan Lucena                     georaster, oci
```

Comment:
@IvanLucena `ilucena` username is no longer valid for GitHub and it should read `IvanLucena` doesn't it?